### PR TITLE
Admin: Fix picotable sorting bug

### PR DESCRIPTION
**Pull request by shawnadelic**
Fix picotable bug when sorting querysets for translated fields.

Refs SHOOP-2398

### CODE REVIEW

**Comment by tulimaki** [comment]
There is few problems in test. Populate if required only populates products with one language. This is issue since we faced problems most of the time only after multiple translations. Also you are not testing the actual issue here which was the duplicate items.

File: shoop/admin/utils/picotable.py [hunk 1]
```diff
@@ -237,11 +238,9 @@ def to_json(self, context=None):
...
     def sort_queryset(self, queryset, desc=False):
         order_by = ("-" if desc else "") + self.sort_field
-        queryset = queryset.order_by(order_by)
-        if self.sort_field.startswith("translations__"):
-            # Ref http://archlinux.me/dusty/2010/12/07/django-dont-use-distinct-and-order_by-across-relations/
-            queryset = queryset.annotate(_dummy_=Count(self.sort_field))
-        return queryset
+        if "translations__" in self.sort_field:
+            queryset = queryset.translated(get_language())
+        return queryset.order_by(order_by)
```
Comment (suutari-ai) [inline shoop/admin/utils/picotable.py]: This creates another bug, since now some rows will be missing from the result. I.e. those that don't have a translation for currently active language.
